recipes/annotsv/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "annotsv" %}
+{% set version = "3.3.4" %}
+{% set sha256 = "9fa4164e7df33304969bd595659995f413b09dff77325a61d73bfe4a020409d3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/lgmgeo/AnnotSV/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - make
+  host:
+    - python >=3.8
+  run:
+    - tk >=8.5
+    - bedtools >=2.25
+    - bcftools >=1.10
+    - openjdk >=8
+    - python >=3.8
+    - coreutils
+    - unzip
+    - pyfaidx >=0.7.1
+    - natsort >=7.1.1
+    - pandas >=1.5.2
+    - tqdm >=4.64.1
+
+# Tests AnnotSV, the annotations install script and variantconvert (needed for VCF outputs)
+test:
+  commands:
+    - AnnotSV -help
+    - bash -n INSTALL_annotations.sh
```

Comment:
```suggestion
    - which INSTALL_annotations.sh
```

Reply:
As-is, you're trying to do a syntax check on `INSTALL_annotations.sh` in the PWD, and it's not there. Suggest just checking that the script is in the path (assuming there's not a `-help`).

Reply:
Yes I was also checking if the script has permissions to run (which is why I chose for `bash -n`)

Reply:
Huh, I didn't even realise that would work for something in the PATH :-D